snippets/resolver/named.conf

```
@@ -1,28 +1,48 @@
 options {
-  directory "/var/cache/bind";
-  listen-on { any; };
-  listen-on-v6 { any; };
-  pid-file "/run/named/named.pid";
+directory "/var/cache/bind";
+allow-recursion {
+192.168.0.0/24;
+127.0.0.1/32;
```

Fix indentation:

```suggestion
  directory "/var/cache/bind";
  allow-recursion {
    192.168.0.0/24;
    127.0.0.1/32;
```

Also, not sure the IPs should be those. `127.0.0.1/32` makes sense, but I think `192.168.0.0/24` should be whatever the subnet the resolver should be accessible on is (which when using docker-compose would be `10.0.10.0/24`, as configured in `docker-compose.yml`). In all cases, `localnets` is probably worth a try and should do the right thing.

---

Ack